config/config.go

```diff
@@ -959,7 +958,7 @@ func NewConfig() (c *Config) {
 				MetricsOutbound:   MetricsDefaults{},
 				MetricsPerRefresh: "1m",
 				Namespaces:        make([]string, 0),
-				RefreshInterval:   "60s",
+				RefreshInterval:   "1m",
```

Comment:
I don't think this matters, it should be an equivalent duration. But agree it should match the documented default.